info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 1    | 1.2.1    | PACKAGE_NAME_INCORRECT_CASE               | Check: warns if a package name is incorrect (non-lower) case<br>Fix: automatically update the case in package name                                                                                                                                          | yes | no                                                                                                                                                                                                                         | Recursively update all imports in the project.                                                                                                                                                                                |
+| 1    | 1.2.1    | PACKAGE_NAME_INCORRECT_PREFIX             | Check: warns if a package name does not start with the company's domain<br>Fix: automatically update the prefix in the package name                                                                                                                            | yes | no                                                                                                                                                                                                                         | Fix the directory where the code is stored.<br>Recursively update all imports in the project.                                                                                                                                 |
+| 1    | 1.2.1    | PACKAGE_NAME_INCORRECT_SYMBOLS            | Check: warns if a package name has incorrect symbols like underscore or non-ASCII letters/digits.Exception: underscores that are used for differentiating of keywords in a name.<br>Fix: no fix currently; will be suggested later                              | no  | no                                                                                                                                                                                                                         | Add autofix for at least converting underscore to a dot or replacing itFix the directory where the code is stored.Cover autofix with tests                                                                                    |
```

Comment:
"..replacing itFix the directory where the code is stored." 
What does it mean?

Reply:
This is the last column, not intended to be read by users. So please just fix typos where appropriate, add a space after punctuation marks, and let's leave it as-is.

Reply:
ok